### PR TITLE
Use MPSCNNConvolutionTranspose in iOS 11.3.1 +

### PR DESCRIPTION
**Pull request by mats-claassen**
There seems to be an issue in iOS 11.3.1 which makes several of our shaders break. This is a workaround for that

### CODE REVIEW

File: Sources/Adapters/Tensorflow/TFConverter+Mappers.swift [hunk 1]
```diff
@@ -101,8 +101,14 @@ public extension TFConverter {
             kernelHeight: weightData.weightShape.kernelHeight,
             strideX: Int(strides.x),
             strideY: Int(strides.y))
+
+        var weights = weightData.weights
+        if #available(iOS 11.3.1, *) {
```
Comment (bryant1410) [inline Sources/Adapters/Tensorflow/TFConverter+Mappers.swift]: What about adding a param to `ConvTranspose` to tell to use MPS' transpose conv, so the condition to use it (`#available(iOS 11.3.1, *)`) is located only in one place?

Comment (mats-claassen) [reply Sources/Adapters/Tensorflow/TFConverter+Mappers.swift]: The #available needs to be there for the code to compile

Comment (bryant1410) [reply Sources/Adapters/Tensorflow/TFConverter+Mappers.swift]: Yeah, but what about checking it only in one place and saving it, like `useMPSConv = #available(iOS 11.3.1, *)`?

Comment (bryant1410) [reply Sources/Adapters/Tensorflow/TFConverter+Mappers.swift]: Oh, good to know!

**Pull request merged**